net.x_y.ykhwong.dosbox-daum.desktop

```diff
@@ -0,0 +1,4 @@
+[Desktop Entry]
+Name=Dosbox Daum
+Type=Application
+Exec=dosbox
```

Comment:
this is insufficient. Missing a few things like `GenericName`, `Comment`, and `Categories`. In addition, `Icon` as I mentionned below.

Reply:
is that sufficient now?